sbt-dotty/sbt-test/sbt-dotty/example-project/test

```diff
@@ -1,4 +1,5 @@
 > run
+> doc
```

Comment:
This is not run on PRs. Did you run it locally? I'll take your words for it 😄

Reply:
Works fine on my machine at least.